### ISSUE TITLE
prebuilt macOS binary doesn't work

### ISSUE THREAD

**Issue by kevinsjoberg** [vamolessa/pepper#22]
Hi 👋

I just purchased a prebuilt binary of pepper and tried to run it on macOS 12.0.1. Unfortunately, it doesn't work. Pepper immediately crashes upon starting it. No output whatsoever.

- `pepper version 0.19.1`
- `macOS 12.0.1`

**Comment by vamolessa** [comment]
Oh, sorry about that.
Is there a `pepper_crash.txt` in the folder from where you tried to run it?
Would you mind pasting it here?
Thanks

**Comment by kevinsjoberg** [comment]
No worries. Ah, nice. Completely missed that this file got generated. Here's the contents.

```console
$ cat /Users/kevinsjoberg/code/Odin/pepper-crash.txt
panicked at 'could not add event', pepper/src/platforms/bsd.rs:100:13
```

**Comment by vamolessa** [comment]
I think I've managed to fix it on `v0.19.3`.
It turns out we can't poll on `/dev/tty` with `kqueue` on macos specifically.
Will close for now. If there's still an issue, please fell free to reopen it.
Thanks for your patiance and for reporting this!
